keywords/patch.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
 var addKeyword = require('./add_keyword');
-var jsonPatch = require('fast-json-patch/src/json-patch');
+var jsonPatch = require('fast-json-patch');
 
 module.exports = function(ajv) {
-  addKeyword(ajv, '$patch', jsonPatch, {
+  addKeyword(ajv, '$patch', jsonPatch, 'applyPatch', {
```

Comment:
you can pass `jsonPatch.applyPatch` here and `jsonMergePatch.apply` above. Then in add_keyword you would just need to use `jsonPatch(null, source, patch, true);` without any additional parameters.